01/Swap.h

```diff
@@ -2,10 +2,6 @@
 
 namespace samples
 {
-	// TODO
-	// Complete Swap() method in lecture slide (p59)
-	// Call Swap() in CallReferenceExample();
-	// Print the numbers after swapping
 	void CallReferenceExample();
-	void Swap(int& number1, int& number2);
+	void SwapMain(int& number1, int& number2);
```

Comment:
i meant.. this function is still be

Swap

But wanted to unify the name patterns for the functions that main() function calls.

CallReferenceExample --> SwapExample()
ReverseInputString --> ReverseInputStringExample()
PrintEverything --> PrintEverythingExample()
AddIntegers --> AddintegersExample()